src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol_Bases.cs

```diff
@@ -111,6 +111,18 @@ protected override void CheckBase(DiagnosticBag diagnostics)
 
                 localBase.CheckAllConstraints(DeclaringCompilation, conversions, location, diagnostics);
             }
+
+            // Records can only inherit from other records or object
+            if (IsRecord && localBase.SpecialType != SpecialType.System_Object && !localBase.IsRecord)
+            {
+                var baseLocation = FindBaseRefSyntax(localBase);
+                diagnostics.Add(ErrorCode.ERR_BadRecordBase, baseLocation);
```

Comment:
>diagnostics.Add(ErrorCode.ERR_BadRecordBase, baseLocation); [](start = 16, length = 59)

I think this is going to break some tests added in [https://github.com/dotnet/roslyn/pull/44876](https://github.com/dotnet/roslyn/pull/44876)

Reply:
> I think this is going to break some tests added in [https://github.com/dotnet/roslyn/pull/44876](https://github.com/dotnet/roslyn/pull/44876)


I'll try to preemptively fix this

---
In reply to: [436047275](https://github.com/dotnet/roslyn/pull/44852#discussion_r436047275) [](ancestors = 436047275)